api/src/shared/config.js

```diff
@@ -450,6 +450,9 @@ const configuration = (function () {
     temporaryStorageForEmailValidationDemand: {
       expirationDelaySeconds: ms(process.env.EMAIL_VALIDATION_DEMAND_TEMPORARY_STORAGE_LIFESPAN || '3d') / 1000,
     },
+    temporaryStorageForAnonymousUserToken: {
```

Comment:
Suggestion pour être plus explicite sur ce que va contenir ce temporary storage (et au passage `temporaryStorageForEmailValidationDemand` aurait dû être nommé `temporaryStorageForEmailValidationDemands`) : 
```suggestion
    temporaryStorageForAnonymousUserTokens: {
```